build.gradle.kts

```diff
@@ -113,6 +117,7 @@ tasks {
     }
 
     test {
+        environment("BUF_CACHE_DIR", File(project.projectDir.path + "/src/test/resources/testData/cachev1").absolutePath)
```

Comment:
Ensures we run with a stable cache dir instead of relying on downloading modules at runtime.